src/components/ListLink/index.js

```diff
@@ -0,0 +1,23 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core'
+import styles from './ListLink.styles'
+import Anchor from 'components/Anchor'
+import Text from 'components/Text'
+import { ReactComponent as Arrow } from 'static/icons/arrow.svg'
+import { TEXT_TYPE } from 'components/Text/constants'
+
+export const ListLink = ({ href, text, title }) => (
+  <Anchor css={styles.anchor} href={href}>
```

Comment:
This component seemed more like a link to me after I started working on it, but let me know if you think I should change this to a `button` with an `onClick` prop instead.